src/apify_client.ts

```diff
@@ -192,10 +192,12 @@ export class ApifyClient {
         ow(id, ow.string.nonEmpty);
         ow(options, ow.object.exactShape({
             clientKey: ow.optional.string.nonEmpty,
+            timeout: ow.optional.number,
         }));
 
         const apiClientOptions = {
             id,
+            timeout: options.timeout,
```

Comment:
this seems wrong, it should be part of the second parameter only, that is how you use it in `RequestQueueClient`